.travis.yml

```diff
@@ -44,17 +50,24 @@ before_install:
     - git fetch --tags
 
 install:
-
     - conda create --name cta-dev python=$TRAVIS_PYTHON_VERSION
     - conda env update -n cta-dev --file environment.yml
     - source activate cta-dev
     - ulimit -s 16000  # increase stack size limit, for libhessio
     - pip install travis-sphinx
     - pip install codecov
     # ----- SST1M:
-    - conda install numpy protobuf libgcc
+    - conda install protobuf libgcc
```

Comment:
should we just add this to environment.yml?  I think we can also add pip packages there for SST-1m (under a `pip` heading).  That way everyone who installs the package using environment.yml will get TargetIO and SST1m support (maybe that's not necessary, though)

Reply:
This is not a good approach for the Target reader, the target_software repository only exists so TravisCI can perform the tests, no user should rely on these pre-compiled binaries. Instead they should install the TARGET libraries from source following the instructions at https://forge.in2p3.fr/projects/gct/wiki/Installing_CHEC_Software (which involves obtaining the latest source files from the cta svn, and performing cmake with swig on the three individual libraries, which also have dependencies on each other).

I have no idea how to make the TARGET libraries "conda installable" due to their complexity and the installation procedure involved, but I imagine it would be a huge undertaking that I do not have time for at the moment.